docs/usage/integration.md

```diff
@@ -0,0 +1,413 @@
+# Integration
+
+TigerBeetle is designed to guard against bugs not only in its [own code](../TIGER_STYLE.md), but
+at the boundaries, in the application code which interfaces with TigerBeetle.
+This is exhibited by the client's API design, which may be [surprising](#retries) when contrasted
+with a more conventional database.
+
+Strict consistency guarantees at the database level simplify application logic and error handling
+farther up the stack.
+
+
```

Comment:
No need for more than two newlines here (there are 4?)

Reply:
Yeah, I do that deliberately, it helps me keep the sections clearly separate in my head as I jump around & edit. It won't show up in the rendered markdown.